.github/workflows/build-and-publish-adhoc.yml

```diff
@@ -1,40 +0,0 @@
-name: Publish ad-hoc image to Docker Hub
```

Comment:
Did you intend to delete this file?  I didn't see any mention of deleting this workflow in the PR - is it still useful?

Reply:
Yes, it's not used anymore